onnx2pytorch/operations/unsqueeze.py

```diff
@@ -20,6 +20,6 @@ def forward(self, data: torch.Tensor, axes: torch.Tensor = None):
         elif isinstance(dims, int):
             return torch.unsqueeze(data, dim=dims)
         else:
-            for dim in sorted(dims, reverse=True):
```

Comment:
Why the change? Was it wrong before?

Reply:
Yes, prior to the change, it was failing `test_unsqueeze_unsorted_axes` (which is taken from the same example in the ONNX operator readme).